tests/components/airnow/conftest.py

```diff
@@ -62,6 +61,4 @@ async def setup_airnow_fixture(hass, config, mock_api_get):
     with patch("pyairnow.WebServiceAPI._get", mock_api_get), patch(
```

Comment:
Are we sure that this is patching in the right place?

Reply:
I'm not sure, I'll double check.

Reply:
The patching was indeed strange / duplicated.
Fixed now.